code-scanning/dependency-review.yml

```
@@ -32,8 +32,8 @@ jobs:
       - name: 'Dependency Review'
         uses: actions/dependency-review-action@v4
         # Commonly enabled options, see https://github.com/actions/dependency-review-action#configuration-options for all available options.
-        # with:
+        with:
+          comment-summary-in-pr: always
         #   fail-on-severity: moderate
-        #   deny-licenses: GPL-1.0-or-later, LGPL-2.0-or-later
-        #   comment-summary-in-pr: true
+        #   deny-licenses: GPL-1.0-or-later, LGPL-2.0-or-later        
```

```suggestion
        #   deny-licenses: GPL-1.0-or-later, LGPL-2.0-or-later
```

Linter failing: trim trailing whitespace.................................................Failed
- hook id: trailing-whitespace

---

we should use SARIF linter integrations :D

---

gah, something keeps inserting extra whitespace, we had fixed that in the last merge :|